src/components/tag/index.tsx

```diff
@@ -2,10 +2,14 @@ import { IconX16 } from '@hashicorp/flight-icons/svg-react/x-16'
 import { TagProps } from './types'
 import s from './tag.module.css'
 
-const Tag = ({ text, onRemove }: TagProps) => {
+const Tag = ({ text, onRemove, removeButtonAriaLabel }: TagProps) => {
```

Comment:
suggestion: we should be able to set the default value of `removeButtonAriaLabel` even when it uses another prop:

```tsx
const Tag = ({ text, onRemove, removeButtonAriaLabel = `Dismiss ${text}` }: TagProps) => {
  // ...
}
```